tests/test_idutils.py

```diff
@@ -32,13 +32,16 @@
     ('http://www.example.org/ark:/13030/tqb3kh97gh8w', ['ark', 'url'], '', ''),
     ('10.1016/j.epsl.2011.11.037', ['doi', 'handle'],
         '10.1016/j.epsl.2011.11.037',
-        'http://dx.doi.org/10.1016/j.epsl.2011.11.037'),
```

Comment:
We still support validating/detecting the old style DOIs. All of these are valid:

```python
10.1016/j.epsl.2011.11.037
doi:10.1016/j.epsl.2011.11.037
http://dx.doi.org/10.1016/j.epsl.2011.11.037
https://dx.doi.org/10.1016/j.epsl.2011.11.037
http://doi.org/10.1016/j.epsl.2011.11.037
https://doi.org/10.1016/j.epsl.2011.11.037
```

However the link generation is changed to generate ``https://doi.org/10.1016/j.epsl.2011.11.037`` instead of ``http://dx.doi.org/10.1016/j.epsl.2011.11.037`` for all of above DOIs.

This is an incompatible change (hence also the v0.2.0), but not a major one. Normally you should be storing the normalised version, and just use the generated link when you need to expose the link.